README.md

```diff
@@ -1,148 +1,91 @@
-# Phoenics
+# Data-driven experiment planning strategies 
```

Comment:
Is it the same readme inside private repo?

Reply:
Will be updated after MR is approved